sshkey_paramiko.r2py

```diff
@@ -94,9 +94,6 @@ class _sshkey_paramiko_BER(object):
   def __init__(self, content=''):
       self.content = content
       self.idx = 0
-
-  def decode(self):
```

Comment:
I think it makes sense to add a comment to explain why you changed this.  The reason is that at some point in the future, if we want to re-port the Paramiko library, we should know why this edit was done.